### PR TITLE
Add Arne Bahlo to 2024

### PR DESCRIPTION
**Pull request by bahlo**

### CODE REVIEW

File: 2024.html [hunk 1]
```diff
@@ -58,6 +58,7 @@ <h3>Kudos to these websites who <!-- @@ got --> plan to get naked in 2024!</h3>
               Thank you!
             -->
             <li>alifeee's <a href="https://alifeee.co.uk/">website</a>, <a href="https://blog.alifeee.co.uk/">blog</a>, and <a href="https://weeknotes.alifeee.co.uk/">weeknotes</a></li>
+            <li><a href="https://arne.me">Arne Bahlo</a></li>
```
Comment (j9t) [inline 2024.html:61]: Minor: Please add leading slash in consistency with other entries. Thanks!

**Review by j9t — CHANGES_REQUESTED** [review]

File: 2024.html (hunk 1 above)
Comment (bahlo) [reply 2024.html]: Done!

**Comment by arkhi** [comment]
@j9t: Since you requested a change, you’re the only one who can merge the PR. I’ll let you process it when you can or I can cancel the change request if you’re short on time.

**Review by j9t — APPROVED** [review]

**Comment by j9t** [comment]
LGTM—thanks both of you!

**Comment by bahlo** [comment]
fyi you can [dismiss a pull request review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/dismissing-a-pull-request-review) and then merge it 😁 

thank you both!

**Pull request merged**